packages/bbui/src/FancyForm/FancyForm.svelte

```diff
@@ -22,9 +25,17 @@
     })
     return valid
   }
+
+  const styles = () => {
```

Comment:
Why is this a function? Could this not be an object?

Reply:
it was just for ease of readability - but it could probably be an array thats just joined, just didn't want to create a massive string all in one go thats hard to understand